Source/NSMenu.m

```diff
@@ -1860,6 +1889,41 @@ - (void) _showOnActivateApp: (NSNotification*)notification
   }
 }
 
+- (void) windowDidChangeScreen: (NSNotification*)notification
+{
+  NSWindow *window = [notification object];
+  NSRect   frame;
+  NSRect   oldScreenFrame;
+  NSRect   newScreenFrame;
+  CGFloat  yOffset;
+
+  if ([window isKindOfClass: [NSMenuPanel class]]
```

Comment:
Could we merge these two if statements into one?

Reply:
Sure, done.